README.md

```diff
@@ -10,6 +10,11 @@ Spoke was created by Saikat Chakrabarti and Sheena Pakanati, and is now maintain
 
 This is generated from [react-apollo-starter-kit](https://github.com/saikat/react-apollo-starter-kit).  Look at that project's README for info on some of the libraries used.
 
+## Deploy to Heroku
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/shakalee/Spoke)
```

Comment:
don't forget to change `shakalee` here at the end.

Reply:
yea - i cant figure out a way to test this without master having an app.json so i'm using my fork to test for now.